src/pages/[platform]/build-a-backend/graphqlapi/connect-api-to-existing-database/index.mdx

```diff
@@ -0,0 +1,834 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Connect API to existing MySQL or PostgreSQL database',
+  description:
+    'Learn how to connect your API to an existing MySQL or PostgreSQL database.',
+    platforms: ['javascript', 'react', 'nextjs', 'angular', 'vue', 'react-native', 'flutter', 'swift', 'android']
```

Comment:
@chrisbonifacio Should this be JS only? Flutter, Swift and Android do not have CDK set up options available on their set up page as noted as a requirement in the first callout.